website/src/content/concepts/merchant-center-proxy-router.mdx

```diff
@@ -11,35 +11,35 @@ https://mc.<cloud-region>.commercetools.com/:projectKey/dashboard
 https://mc.<cloud-region>.commercetools.com/:projectKey/products
 https://mc.<cloud-region>.commercetools.com/:projectKey/orders
 https://mc.<cloud-region>.commercetools.com/:projectKey/avengers
-...
 ```
 
-This way it looks like you are using one single web application even though underneath there are multiple Custom Applications.
+This way, it looks like you're using a single web application, even though there are multiple Custom Applications underneath. Let's understand how this works.
 
-# Routing
+# Server-side routing
 
-Every initial request to render the page goes through a server component called **Merchant Center Proxy Router**.<br />
-This server is primarily responsible for matching the incoming request to the appropriate Custom Application and to forward the request to the actual URL location of the Custom Application. The response then should be serving the `index.html` of the Custom Application.
-
-For example, the user tries to access the `/:projectKey/products` URL. The request goes through the Proxy Router which successfully matches the `products` identifier. The Proxy Router then loads the related configuration for the Products application (assuming that the application is installed for the given `projectKey`) and forwards the request to the location where the Custom Application is hosted. The `index.html` is then returned as a response and the Proxy Router forwards that to the browser. The Products application renders.
+When the user accesses the Merchant Center through the browser, the request to render the page goes through a server component called **Merchant Center Proxy Router**.<br />
+This server is primarily responsible for matching the incoming request to the appropriate Custom Application using the `entryPointUriPath` ([the identifier of the Custom Application](/api-reference/application-config#entrypointuripath)). The request then is [forwarded](#forwarding-requests) to the actual [URL location of the Custom Application](/api-reference/application-config#envproductionurl), which responds with the `index.html`.
 
 <Info>
 
-A Custom Application is a Single-Page Application that uses client-side routing. Therefore, you need to instruct your hosting provider to rewrite all requests to serve the `index.html`.
+A Custom Application is a Single-Page Application that uses client-side routing. So, you need to configure your hosting provider to rewrite all requests to serve the `index.html`. Conceptually it would mean something like this: `/* --> /index.html`. See [forwarding requests](#forwarding-requests) for more detailed information.
```

Comment:
```suggestion
A Custom Application is a Single-Page Application that uses client-side routing, so you need to configure your hosting provider to rewrite all requests to serve the `index.html`. Conceptually it would mean something like this: `/* --> /index.html`. See [forwarding requests](#forwarding-requests) for more detailed information.
```